venues/ICML.cc/2017/MLAV/python/admin-init.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+
+import openreview
+import sys, os
+import config
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "/Users/mandler/projects/openreview-scripts/utils"))
```

Comment:
Appears to have pulled it from <<UTILS_DIR>>.  Where do environmental vars hide on a Mac?
In the meantime I'll change it in the code.